engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -77,6 +76,11 @@
     public static final String DUMP_SHADERS = "DumpShaders";
     public static final String VOLUMETRIC_FOG = "VolumetricFog";
 
+    public static final SimpleUri WINDOW_POS_X_ = new SimpleUri("engine:windowPosX");
+    public static final SimpleUri WINDOW_POS_Y_ = new SimpleUri("engine:windowPosY");
```

Comment:
I think both of these can be put together into a single `Setting<Vector2i>` for easier access. However, this change will break things, so it may be better off in a separate PR.

Reply:
I actually favour this change, for reasons other than easier access. Rather than subscribe to either width *and* height, you can simply subscribe to one property. So I think let's make it a `Vector2i` in this PR, unless someone has a problem. This PR will be kinda messy and break-all-the-things either way.